p2panda-net-next/src/actors/iroh/endpoint.rs

```diff
@@ -141,6 +159,19 @@ impl ThreadLocalActor for IrohEndpoint {
                     .bind()
                     .await?;
 
+                // Watch for changes of our own endpoint address.
```

Comment:
We may want to split this listener out at some point, since the address updates will need to be communicated to several sub-systems (for example, they will be used to notify the user via eventually consistent topic streams).